app/views/shared/_item_list.html.erb

```diff
@@ -1,3 +1,4 @@
+<h1><%= @content_header %></h1>
```

Comment:
and we could have the category heading on the category show page instead of the partial